### PR TITLE
Fix nodes.stats

### PR DESCRIPTION
**Pull request by Anaethelion**
Incomplete but most of the heavy lifting seems ok.
Will do a second part for the special cases left!

### CODE REVIEW

**Review by swallez — COMMENTED** [review]
Some fields are still required. Should we make them all optional considering that filtering is a first class feature of this API?

File: specification/nodes/_types/Stats.ts [hunk 1]
```diff
@@ -65,22 +142,52 @@ export class KeyedProcessor {
 }
 
 export class AdaptiveSelection {
-  avg_queue_size: long
-  avg_response_time: long
+  avg_queue_size?: long
+  avg_response_time?: long
   avg_response_time_ns: long
```
Comment (swallez) [inline specification/nodes/_types/Stats.ts:147]: This field (and some others in other classes) is left required. Is this intentional?

**Review by delvedor — APPROVED** [review]
LGTM

**Comment by Anaethelion** [comment]
I'll rebase the main branch to resolve conflicts!

**Pull request merged**